src/main/java/seedu/address/logic/commands/edit/EditExamCommand.java

```diff
@@ -0,0 +1,222 @@
+package seedu.address.logic.commands.edit;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DAY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LINK;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VENUE;
+
+import java.util.List;
+import java.util.Optional;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.GuiState;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.module.Link;
+import seedu.address.model.module.ModBookDate;
+import seedu.address.model.module.Module;
+import seedu.address.model.module.Timeslot;
+import seedu.address.model.module.Venue;
+import seedu.address.model.module.exam.Exam;
+import seedu.address.model.module.exam.ExamName;
+
+public class EditExamCommand extends EditCommand {
+    public static final String MESSAGE_DUPLICATE_EXAM = "This exam already exists.";
+    public static final String MESSAGE_EDIT_EXAM_SUCCESS = "Edited Exam %s in Module %s";
+    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits an exam in the Mod book. "
+            + "\nParameters: INDEX "
+            + PREFIX_CODE + "MOD_CODE "
```

Comment:
now doesn't need code right?

Reply:
Good catch. I realised I was referencing rohit's PR and his didn't change either ah. I'll make the changes for his files too